File: HelpDesk/Models/Ticket.cs
using src;

public class Ticket
{
    public int Id { get; set; }
    public string Title { get; set; } 
    public bool Open { get; set; }
    public string Description { get; set; }
    public int UserId { get; set; }
    public virtual User User { get; set; }
}

File: UI/src/app/models/Ticket.ts
import { UserDTO } from "./User";

export interface TicketBody {
    id: number;
    userid: number | null;
    title: string;
    open: boolean;
    description: string;
}

export interface TicketDTO {
    id: number;
    userid: number | null;
    title: string;
    open: boolean;
    description: string;
}
